extension/src/popup/helpers/searchAsset.ts

```diff
@@ -21,6 +21,32 @@ export const searchAsset = async ({
   }
 };
 
+export const getNativeContractDetails = (networkDetails: NetworkDetails) => {
+  const NATIVE_CONTRACT_DEFAULTS = {
+    code: "XLM",
+    decimals: 7,
+    domain: "https://stellar.org",
+    icon: "",
+    org: "",
+  };
+  switch (networkDetails.network as keyof typeof NETWORKS) {
+    case NETWORKS.PUBLIC:
+      return {
+        ...NATIVE_CONTRACT_DEFAULTS,
+        contract: "CAS3J7GYLGXMF6TDJBBYYSE3HQ6BBSMLNUQ34T6TZMYMW2EVH34XOWMA",
+        issuer: "GDMTVHLWJTHSUDMZVVMXXH6VJHA2ZV3HNG5LYNAZ6RTWB7GISM6PGTUV",
+      };
+    case NETWORKS.TESTNET:
+      return {
+        ...NATIVE_CONTRACT_DEFAULTS,
+        contract: "CDLZFC3SYJYDZT7K67VZ75HPJVIEUVNIXF47ZG2FB2RMQQVU2HHGCYSC",
+        issuer: "",
```

Comment:
curious why there's an issuer on mainnet but not testnet?

Reply:
When Testnet resets, my understanding is that this contract will need to be redeployed, at which point the issuer could be different. I believe this value will be out of date eventually.

We don't actually do anything with this value, so this shouldn't pose a problem. The `issuer` key is more useful for trustlines, I'm just following the common interface here